### PR TITLE
Create conversation with message in public API

### PR DESCRIPTION
**Pull request by lasryaric**

### CODE REVIEW

**Review by spolu — APPROVED** [review]

File: front/pages/api/v1/w/[wId]/assistant/conversations/index.ts [hunk 1]
```diff
@@ -83,12 +86,29 @@ async function handler(
...
 
       const conversation = await createConversation(auth, {
         title,
         visibility,
       });
+
+      if (message) {
+        // Not awaiting this promise on prupose. We want to answer "OK" to the client ASAP and process
+        // the events in the background.
+        void postUserMessageWithPubSub(auth, {
```
Comment (philipperolet) [inline front/pages/api/v1/w/[wId]/assistant/conversations/index.ts:99]: we have changed this to await for user_new_message in postMessage
We should do that here too probably?

**Review by philipperolet — COMMENTED** [review]
LGTM save the await re the discussion we had early this week

File: front/pages/api/v1/w/[wId]/assistant/conversations/index.ts (hunk 1 above)
Comment (spolu) [reply front/pages/api/v1/w/[wId]/assistant/conversations/index.ts:99]: Not here as this is a conversation creation endpoint. This is just a way to bundle the new message in the same call

Comment (spolu) [reply front/pages/api/v1/w/[wId]/assistant/conversations/index.ts:99]: This non-waiting is aligned with the conversation creation with message imlpementation in our internal API

**Pull request merged**